docs/MainSite/ReportingGuide.md

```diff
@@ -0,0 +1,91 @@
+law enforcement first. If you are unsure what law enforcement agency
+is appropriate, please include this in your report and we will attempt
+to notify them.
+
+Reports of violations of the code of conduct can be as formal or
+informal as needed for the situation at hand. If possible, please
+include as much information as you can. If you feel comfortable,
+please consider including:
+- Your contact info (so we can get in touch with you if we need to
+  follow up).
+- Names (real, nicknames, or pseudonyms) of any individuals
+involved. If there were other witnesses besides you, please try to
+include them as well.
+- When and where the incident occurred. Please be as specific as
+  possible.
+- Your account of what occurred. If there is a publicly available
+record (e.g. a mailing list archive or Slack logs) please include a
+link.
+- Any extra context you believe existed for the incident.
+- If you believe this incident is ongoing.
+- Any other information you believe we should have.
+
+### What happens after you file a report?
+
+You will receive an email from the Code of Conduct Committee or
+project leader(s) you contacted, acknowledging receipt within 24 hours
+(and we will aim to respond much quicker than that). They will review
+the incident and try to determine:
+- What happened and who was involved.
+- Whether this event constitutes a code of conduct violation.
+- Whether this is an ongoing situation, or if there is a threat to
+  anyone’s physical safety.
+
+Once the Code of Conduct Committee or contacted project leader(s) have
+a complete account of the events they will make a decision on how to
+respond, possibly in consultation with the full Executive Committee
+(excluding anyone involved in the incident). Responses may include:
+- Nothing, if we determine no violation occurred or it has already
+  been appropriately resolved.
+- Providing either moderation or mediation to ongoing interactions
+  (where appropriate, safe, and desired by both parties).
+- A private reprimand from the working group to the individuals
+  involved.
+- An imposed vacation (i.e. asking someone to “take a week off” from a
+  mailing list or Slack).
+- Escalation to the appropriate institutions.
+- Involvement of relevant law enforcement if appropriate.
+
+If the situation is not resolved within one week, we’ll respond within
+one week to the original reporter with an update and explanation.
+Once we’ve determined our response, we will separately contact the
+original reporter and other individuals to let them know what actions
+(if any) we’ll be taking. We will take into account feedback from the
+individuals involved on the appropriateness of our response, but we
+don’t guarantee we’ll act on it.
+
+### Code of Conduct Committee
+
+The current members of the Code of Conduct Committee (and the only
+people that see emails sent to conduct@spectre-code.org) are:
+- TBD: Saul Teukolsky
+- TBD: Harald Pfieffer
+- TBD: ???
```

Comment:
I volunteer to be on this committee

Reply:
Thank you for volunteering, but I think the members of this committee should be people who are familiar with SpECTRE, but rarely code.